go/engine/wot_attest.go

```diff
@@ -0,0 +1,177 @@
+// Copyright 2020 Keybase, Inc. All rights reserved. Use of
+// this source code is governed by the included BSD license.
+
+package engine
+
+import (
+	"crypto/hmac"
+	"crypto/sha256"
+	"encoding/hex"
+	"encoding/json"
+	"errors"
+	"strings"
+
+	"github.com/keybase/client/go/libkb"
+	keybase1 "github.com/keybase/client/go/protocol/keybase1"
+	jsonw "github.com/keybase/go-jsonw"
+)
+
+type WotAttestArg struct {
+	Attestee     keybase1.UserVersion
+	Confidence   keybase1.Confidence
+	Attestations []string
+}
+
+// WotAttest is an engine.
+type WotAttest struct {
+	arg *WotAttestArg
+	libkb.Contextified
+}
+
+// NewWotAttest creates a WotAttest engine.
+func NewWotAttest(g *libkb.GlobalContext, arg *WotAttestArg) *WotAttest {
+	return &WotAttest{
+		arg:          arg,
+		Contextified: libkb.NewContextified(g),
+	}
+}
+
+// Name is the unique engine name.
+func (e *WotAttest) Name() string {
+	return "WotAttest"
+}
+
+// GetPrereqs returns the engine prereqs.
+func (e *WotAttest) Prereqs() Prereqs {
+	return Prereqs{Device: true}
+}
+
+// RequiredUIs returns the required UIs.
+func (e *WotAttest) RequiredUIs() []libkb.UIKind {
+	return []libkb.UIKind{}
+}
+
+// SubConsumers returns the other UI consumers for this engine.
+func (e *WotAttest) SubConsumers() []libkb.UIConsumer {
+	return nil
+}
+
+// Run starts the engine.
+func (e *WotAttest) Run(mctx libkb.MetaContext) error {
+	luArg := libkb.NewLoadUserArgWithMetaContext(mctx).WithUID(e.arg.Attestee.Uid)
```

Comment:
We probably need to do a full ID on the user, which will return a `User`.

Reply:
This can be changed when we handle proofs.